### PR TITLE
Update Lighthouse/Shipyard to devel

### PR DESCRIPTION
**Pull request by dfarrell07**
See commit messages for details.

This update should fix Lighthouse's CI (nginx image breakage).

### CODE REVIEW

**Comment by tpantelis** [comment]
I was trying this locally a few days ago. Because of the `logr` version bump, we need to bump `sigs.k8s.io/controller-runtime` to _v0.11.2_. However this leads to other build errors that should be fixed by https://github.com/submariner-io/admiral/pull/342. Hopefully that will be it.

File: go.mod [hunk 1]
```diff
@@ -10,14 +10,13 @@ require (
 	github.com/fatih/color v1.13.0 // indirect
 	github.com/fsnotify/fsnotify v1.5.1 // indirect
 	github.com/go-errors/errors v1.2.0 // indirect
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.2.0
```
Comment (mkolesnik) [inline go.mod:13]: This gets bumped which I guess isn't backward compatible :/

**Comment by dfarrell07** [comment]
```
[12:52:59.764] [submariner-operator]$ CGO_ENABLED=0 go build -trimpath -ldflags -s -w -X github.com/submariner-io/submariner-operator/pkg/version.Version=devel-838eff9 		   -X github.com/submariner-io/submariner-operator/api/submariner/v1alpha1.DefaultSubmarinerOperatorVersion=devel -o bin/subctl-devel-838eff9-linux-amd64 ./pkg/subctl/main.go
# sigs.k8s.io/controller-runtime/pkg/log
vendor/sigs.k8s.io/controller-runtime/pkg/log/deleg.go:163:2: cannot use res (type *DelegatingLogger) as type logr.Logger in return argument
vendor/sigs.k8s.io/controller-runtime/pkg/log/deleg.go:179:2: cannot use res (type *DelegatingLogger) as type logr.Logger in return argument
vendor/sigs.k8s.io/controller-runtime/pkg/log/deleg.go:195:2: cannot use res (type *DelegatingLogger) as type logr.Logger in return argument
vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:67:26: cannot use NullLogger{} (type NullLogger) as type logr.Logger in argument to Log.Fulfill
vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:82:41: cannot use NullLogger{} (type NullLogger) as type logr.Logger in argument to NewDelegatingLogger
vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:86:6: cannot use Log (type *DelegatingLogger) as type logr.Logger in assignment
vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:88:13: assignment mismatch: 1 variable but logr.FromContext returns 2 values
vendor/sigs.k8s.io/controller-runtime/pkg/log/null.go:30:5: cannot use NullLogger{} (type NullLogger) as type logr.Logger in assignment
vendor/sigs.k8s.io/controller-runtime/pkg/log/null.go:48:32: undefined: logr.InfoLogger
vendor/sigs.k8s.io/controller-runtime/pkg/log/null.go:54:2: cannot use log (type NullLogger) as type logr.Logger in return argument
vendor/sigs.k8s.io/controller-runtime/pkg/log/null.go:54:2: too many errors
make: *** [Makefile:192: bin/subctl-devel-838eff9-linux-amd64] Error 2
```

**Comment by dfarrell07** [comment]
```
controllers/test/driver.go:56:17: invalid operation: cannot compare d.Client == nil (operator == not defined on untyped nil) (typecheck)
	if d.Client == nil {
	               ^
controllers/servicediscovery/servicediscovery_controller_test.go:46:12: t.DoReconcile undefined (type *testDriver has no field or method DoReconcile) (typecheck)
		_, _ = t.DoReconcile()
		         ^
controllers/servicediscovery/servicediscovery_controller_test.go:53:33: t.InitClientObjs undefined (type *testDriver has no field or method InitClientObjs) (typecheck)
				t.InitClientObjs = append(t.InitClientObjs, newDNSConfig(""), newDNSService(clusterIP))
				                            ^
<many more>
```

**Comment by mkolesnik** [comment]
As you can see `controller-runtime` is pinned to an older version, it probably needs to be bumped as well

```
sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.9.0
```

**Comment by dfarrell07** [comment]
This is being done in https://github.com/submariner-io/submariner-operator/pull/2037

**Pull request closed without merging**